content/en/observability_pipelines/rehydration.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Rehydration
+description: Learn about using Rehydration to pull archived logs and processing them in Observability Pipelines.
+disable_toc: false
+private: true
+further_reading:
+- link: "/observability_pipelines/processors/"
+  tag: "Documentation"
+  text: "Learn more about processors"
+- link: "/observability_pipelines/packs/"
+  tag: "Documentation"
+  text: "Learn more about Packs"
+---
+
+{{< callout
+ btn_hidden="true" header="false">}}
+Rehydration is in Preview.
+{{< /callout >}}
+
+## Overview
+
+Rehydration for Observability Pipelines enables you to pull archived logs from object storage and process them in Observability Pipelines, including with [Packs][1]. This gives you consistent access to historical context without having to rebuild workflows or modify ingestion pipelines.
+
+Organizations often store large volumes of logs in cost-efficient, long-term archives to control spend and meet compliance requirements. However, historical data often becomes difficult to access when there is a security incident, audit request, or operational investigation. Retrieving archived logs from cold storage can be slow, manual, and disruptive, requiring ad-hoc scripts, decompression, or dedicated engineering effort. Rehydration for Observability Pipelines solves these issues.
+
+## How Rehydration works
+
+Rehydration provides an automated workflow for retrieving and reprocessing archived logs stored in object stores, such as Amazon S3, Google Cloud Storage, and Azure Blob Storage. This helps you balance storage efficiency with quick access to historical data.
+
+With Rehydration, you can:
```

Comment:
I don't how invested you are in this section, but I think this could be more concise, maybe just a bullet list. A lot of these are already mentioned in the sections above. For example, "Eliminate manual handling" is repeating "Rehydration provides an automated workflow..."